### DISCUSSION TITLE
有没有人用过Rider啊,请教一个换行问题

### DISCUSSION THREAD

**Discussion by yundou147** [BYJRK/DotNet-Discussions#25] [Q&A] [answered]
这是我想要的格式
![image](https://user-images.githubusercontent.com/73925530/218289440-aaa3ea6a-6541-4135-a64d-c5672ddfc4be.png)
ctrl+k+d 整理代码后就变成一行了,我想要上面的结果.有大神知道怎么配置么
![image](https://user-images.githubusercontent.com/73925530/218289458-9f376020-aecf-4224-83f2-ba0c446eee40.png)

**Comment by yundou147** [comment]
@BYJRK

**Reply by BYJRK** [reply]
我没怎么研究过rider的快捷键

**Comment by Organ1sm** [comment, marked as answer]
settings下的code style 里面去改你想要的格式。

**Reply by yundou147** [reply]
就是所有选项都看了,都改不了.

**Comment by kevin100702** [comment]
rider 也有csharpier插件，你可以试试看

**Reply by yundou147** [reply]
好的,谢谢.我去研究一下

**Comment by Organ1sm** [comment]
Lines and wrap下的选项看过了吗？

**Reply by Organ1sm** [reply]
下面有method signature

**Reply by yundou147** [reply]
![image](https://user-images.githubusercontent.com/73925530/218350946-680c1c7d-c925-4def-816f-ee6a4e8692bc.png)
buguan不管这里怎么调整,我格式化后都变成一行了

**Reply by Organ1sm** [reply]
上面的wrap long lines 选了吗

**Reply by yundou147** [reply]
![image](https://user-images.githubusercontent.com/73925530/218359755-eeb66c98-066a-45f0-9a08-32c8fc6c048d.png)
好像这样解决了

**Reply by Organ1sm** [reply]
ok
